PWGGA/GammaConv/AliAnalysisTaskGammaConvV1.cxx

```diff
@@ -764,8 +770,8 @@ void AliAnalysisTaskGammaConvV1::InitBack(){
         fBGHandlerRP[iCut] = NULL;
       } else if(((AliConversionMesonCuts*)fMesonCutArray->At(iCut))->BackgroundHandlerType() != 2){
         fBGHandlerRP[iCut] = new AliConversionAODBGHandlerRP(
-                                  ((AliConvEventCuts*)fEventCutArray->At(fiCut))->IsHeavyIon(),
-                                  ((AliConversionMesonCuts*)fMesonCutArray->At(fiCut))->UseTrackMultiplicity(),
+                                  ((AliConvEventCuts*)fEventCutArray->At(fiCut))->IsHeavyIon(),  // SFS fiCut, is this intended?
```

Comment:
Id assume fiCut is a typo here and it is meant to be iCut. Do you agree?

Reply:
Yes, it has to be iCut - please fix it

Reply:
done